src/style.css

```diff
@@ -0,0 +1,94 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype"),
+    url(./fonts/Avenir-Heavy.ttf) format("truetype"),
+    url(./fonts/Avenir.ttc) format("truetype");
+}
+
+body {
+  margin-top: 0;
```

Comment:
you can reset every margin's, not only top one, if test's allowing you to do that

Reply:
I've done this before. Unfortunately, all 6 tests are not allowed.